packages/ui/atomic/health-check/src/assertions.ts

```diff
@@ -30,7 +30,17 @@ export function ensureDocFile() {
 }
 
 export function ensureConsistentElementName() {
-  throw 'TODO: CDX-1389';
+  ensureDocFile();
+  const {elementName} = getJsonPkg();
+  const {components} = JSON.parse(readFileSync(componentDocPath).toString());
```

Comment:
Instead of asserting that the file must be named `atomic-docs.json`, could we make it more generic and assert that it must be named `stencil-docs.json`? `atomic-docs` makes it sound like we're the ones generating it inside `@coveo/atomic` IMO